populate_db.sql

```diff
@@ -44,9 +45,10 @@
 
 
 -- Create Schemas:
-CREATE SCHEMA erc20;
-CREATE SCHEMA prices;
-CREATE SCHEMA gnosis_protocol_v2;
+CREATE SCHEMA IF NOT EXISTS erc20;
+CREATE SCHEMA IF NOT EXISTS prices;
+CREATE SCHEMA IF NOT EXISTS gnosis_protocol_v2;
+CREATE SCHEMA IF NOT EXISTS dune_user_generated;
```

Comment:
All the if not exists and truncate tables added here are to avoid conflicts on each test run when we try to repopulate data. This is also a step towards specifying specific seed data for different tests later.